cmd/zpool/zpool_main.c

```diff
@@ -4712,6 +4703,32 @@ print_zpool_script_list(char *subcommand)
 	free(sp);
 }
 
+static int
+get_namewidth_iostat(zpool_handle_t *zhp, void *data)
```

Comment:
Since there's only one caller, change `void *data` to `iostat_cbdata_t *cb`.

Reply:
I'd like to but the iterator function expects a `void *` for the data.  So we can't.